src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -117,10 +129,26 @@ export class MdAutocompleteTrigger implements AfterContentInit, OnDestroy {
     } else {
       this.openPanel();
       this._keyManager.onKeydown(event);
+      if (event.keyCode === UP_ARROW || event.keyCode === DOWN_ARROW) {
+        this._scrollToOption();
+      }
     }
   }
 
   /**
+   * Given that we are not actually focusing active options, we must manually adjust scroll
+   * to reveal options below the fold. First, we find the offset of the option from the top
+   * of the panel. The new scrollTop will be that offset - the panel height + the option
+   * height, so the active option will be just visible at the bottom of the panel.
+   */
+  private _scrollToOption(): void {
+    const optionOffset = this._keyManager.activeItemIndex * AUTOCOMPLETE_OPTION_HEIGHT;
+    const newScrollTop =
+        Math.max(0, optionOffset - MD_AUTOCOMPLETE_PANEL_HEIGHT + AUTOCOMPLETE_OPTION_HEIGHT);
```

Comment:
Yikes! Thanks for catching. Must have messed up the renaming option in WebStorm.